.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
Out of curiosity, what is this for?

Reply:
Only node 16 is broken on the CI and its due to a nexus connectivity issue. We're holding off merging until we fix the CI, this was a hopeful shot in the dark that this would get it working. It did not work 😭. I plan on reverting this change when we get closer to merging